### PR TITLE
Atlantic wrap up post

### PR DESCRIPTION
**Pull request by mxsasha**
----
:books: Documentation preview :books:: https://writethedocs-www--2031.org.readthedocs.build/

### CODE REVIEW

File: docs/conf/atlantic/2023/news/thank-you-recap.rst [hunk 1]
```diff
@@ -0,0 +1,123 @@
+:template: {{year}}/generic.html
+:banner: _static/conf/images/headers/venue.png
+
+.. post:: Sept 23, 2023
+   :tags: recap, thanks, {{year}}, {{shortcode}}-{{year}}
+
+Conference highlights and thanks for attending!
```
Comment (plaindocs) [inline docs/conf/atlantic/2023/news/thank-you-recap.rst]:
```suggestion
Conference highlights
```

How do we feel about this?

File: docs/conf/atlantic/2023/news/thank-you-recap.rst [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+Hey everyone,
+
+Our team is now emerging from the post-conference recovery period, and we wanted to send out a quick message to thank everyone for helping to make our conference such a success. 
+We appreciate everyone who makes the conference possible: speakers, sponsors, organizers, volunteers, and attendees alike!
+
+Conference recap
+----------------
+
+At **almost 300 attendees** this conference was a welcoming and friendly event.
```
Comment (plaindocs) [inline docs/conf/atlantic/2023/news/thank-you-recap.rst:18]: "Over" (cos speakers are attendees too) ?

File: docs/conf/atlantic/2023/news/thank-you-recap.rst [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+This was our first Atlantic conference, where we aimed to have a schedule that suited attendees from Europe, Africa, US East Coast and parts of South America.
```
Comment (plaindocs) [inline docs/conf/atlantic/2023/news/thank-you-recap.rst]:
```suggestion
This was our first Atlantic conference, where we intentionally created a schedule that suited attendees from Europe, Africa, US East Coast and parts of South America. In practice, we had attendees for all over the world, including the West coast of the United States and Asia. 
```

File: docs/conf/atlantic/2023/news/thank-you-recap.rst [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+However, we also had attendees from other places around the world.
```
Comment (plaindocs) [inline docs/conf/atlantic/2023/news/thank-you-recap.rst]:
```suggestion
```

File: docs/conf/atlantic/2023/news/thank-you-recap.rst [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+
+We had the usual mix of talks on the main stage along with Q&A, lightning talks, unconference sessions, writing sprints, and a sponsor expo, similar to previous virtual conferences.
```
Comment (plaindocs) [inline docs/conf/atlantic/2023/news/thank-you-recap.rst]:
```suggestion
We had the usual mix of talks on the main stage, along with moderated Q&A, lightning talks, unconference sessions, writing sprints, and a sponsor expo, similar to previous virtual conferences.
```

File: docs/conf/atlantic/2023/news/thank-you-recap.rst [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+
+Talk videos
+-----------
+
+`Videos of the talks`_ are now available on YouTube. This includes all conference talks and Q&A, as well as the lightning talks.
+You can also subscribe to our `YouTube channel`_ so you don’t miss any future content.
```
Comment (plaindocs) [inline docs/conf/atlantic/2023/news/thank-you-recap.rst]:
```suggestion
You can also subscribe to our `YouTube channel`_ so you don’t miss any talk videos from our conferences.
```

File: docs/conf/atlantic/2023/news/thank-you-recap.rst [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+.. _YouTube channel: https://www.youtube.com/writethedocs
+
+Sketchnotes
+-----------
+
+We’re excited to share the `full resolution sketchnotes`_ from the conference on Flickr. You are welcome to use them for non-commercial purposes, like including in your write up from the conference :)
+
+Thanks to `Dennis Dawson`_ for doing them this year!
+
+.. _full resolution sketchnotes: https://flic.kr/ps/
```
Comment (plaindocs) [inline docs/conf/atlantic/2023/news/thank-you-recap.rst]: FIXME

File: docs/conf/atlantic/2023/news/thank-you-recap.rst [hunk 8]
```diff
@@ -0,0 +1,123 @@
...
+Huge thanks also to `Bart at Work`_ for handling the A/V production on our event, it was flawless once again this year.
+
+.. _Videos of the talks: https://youtube.com/playlist?list=PLZAeFn6dfHplddJfvbke1bpUzZGozb2Yj&si=AOwsHHgwGEXqbUm0
+.. _Bart at Work: https://www.bartatwork.com/
+.. _YouTube channel: https://www.youtube.com/writethedocs
+
+Sketchnotes
+-----------
+
+We’re excited to share the `full resolution sketchnotes`_ from the conference on Flickr. You are welcome to use them for non-commercial purposes, like including in your write up from the conference :)
```
Comment (plaindocs) [inline docs/conf/atlantic/2023/news/thank-you-recap.rst]:
```suggestion
We’re excited to share the `full resolution sketchnotes`_ from each talk. You are welcome to use them for non-commercial purposes, like including them in your conference write up ;)
```

File: docs/conf/atlantic/2023/news/thank-you-recap.rst [hunk 9]
```diff
@@ -0,0 +1,123 @@
...
+Thanks to `Dennis Dawson`_ for doing them this year!
+
+.. _full resolution sketchnotes: https://flic.kr/ps/
+.. _Dennis Dawson: https://dennissdawson.wixsite.com/mr--dawson/portfolio
+
+Shirts still on sale
+--------------------
+
+If you want a souvenir from this year's conference, our `Write the Docs Atlantic 2023 Pop-Up Shop <https://shirt.writethedocs.org/>`_ is still open until September 30th!
+We have loose and fitted options from different fabrics, and you can choose exactly the size and cut you want.
```
Comment (plaindocs) [inline docs/conf/atlantic/2023/news/thank-you-recap.rst]:
```suggestion
We have loose and fitted options from different fabrics, and you can choose the size and cut you want.
```

File: docs/conf/atlantic/2023/news/thank-you-recap.rst [hunk 10]
```diff
@@ -0,0 +1,123 @@
...
+
+The campaign will run in 3-day batches, and shipping is available from the USA or Europe.
```
Comment (plaindocs) [inline docs/conf/atlantic/2023/news/thank-you-recap.rst]:
```suggestion
The campaign will run in 3-day batches, with shipping from the USA or Europe.
```

File: docs/conf/atlantic/2023/news/thank-you-recap.rst [hunk 11]
```diff
@@ -0,0 +1,123 @@
...
+
+Come see us at our Australia conference
+---------------------------------------
```
Comment (plaindocs) [inline docs/conf/atlantic/2023/news/thank-you-recap.rst]:
```suggestion
Attend Write the Docs Australia 
-----------------------------------------------
```

File: docs/conf/atlantic/2023/news/thank-you-recap.rst [hunk 12]
```diff
@@ -0,0 +1,123 @@
...
+
+Don’t forget our `Australia conference`_ on **Dec 7-8, 2023**. The CFP is open until September 22nd.
+
+We’re also looking for more sponsors to make our other events just as magical as this year’s Prague event. You can see all our `sponsorship options`_ on our website.
```
Comment (plaindocs) [inline docs/conf/atlantic/2023/news/thank-you-recap.rst]:
```suggestion
We’re also looking for more sponsors to make our other events just as magical as this year’s Atlantic event. You can see all our `sponsorship options`_ on our website.
```

File: docs/conf/atlantic/2023/news/thank-you-recap.rst [hunk 13]
```diff
@@ -0,0 +1,123 @@
...
+
+.. _Australia conference: https://www.writethedocs.org/conf/australia/2023/
+.. _sponsorship options: https://www.writethedocs.org/sponsorship/
+
+Write the Docs Slack
+--------------------
+
+Our Slack network has really grown more this last year. Our #general Slack channel has been extra busy in the days and weeks leading up to the conference, and we hope that y’all continue the conversations throughout the year. If you haven’t joined, now is the time!
```
Comment (plaindocs) [inline docs/conf/atlantic/2023/news/thank-you-recap.rst]:
```suggestion
Our Slack network keeps on growing. Our #general channel in particular has been extra busy in the days and weeks leading up to the conference, and we hope that y’all continue the conversations throughout the year. If you haven’t joined, now is the time! 
```

File: docs/conf/atlantic/2023/news/thank-you-recap.rst [hunk 14]
```diff
@@ -0,0 +1,123 @@
...
+
+.. _monthly newsletter: http://writethedocs.org/newsletter/
+
+Local Meetups
+-------------
+
+We hope you continue to `build your local or online communities`_ and keep in touch with other documentarians. 
+We have a number of meetups in cities across the country, and if you don’t see your city listed consider `starting`_ a meetup!
+
+New this year are our `Quorum Meetups`_ which are virtual and targeting large geographic areas. There’s already one covering most everyone in the US or in a similar timezone.
```
Comment (plaindocs) [inline docs/conf/atlantic/2023/news/thank-you-recap.rst]: They probably aren't new this year?

**Review by plaindocs — CHANGES_REQUESTED** [review]
Looks solid! I've suggested enough tweaks to merit a change request.

File: docs/conf/atlantic/2023/news/thank-you-recap.rst (hunk 2 above)
Comment (mxsasha) [reply docs/conf/atlantic/2023/news/thank-you-recap.rst:18]: Well, depends how you count. 301 in tito (ex speakers), 292 total registrations in hopin.

File: docs/conf/atlantic/2023/news/thank-you-recap.rst (hunk 14 above)
Comment (mxsasha) [reply docs/conf/atlantic/2023/news/thank-you-recap.rst]: Fixed

**Review by plaindocs — APPROVED** [review]
Looks good with a fixed up Flickr URL and a live youtube URL

File: docs/conf/atlantic/2023/news/thank-you-recap.rst [hunk 15]
```diff
@@ -0,0 +1,122 @@
...
+.. _YouTube channel: https://www.youtube.com/writethedocs
+
+Sketchnotes
+-----------
+
+We’re excited to share the `full resolution sketchnotes`_ from each talk. You are welcome to use them for non-commercial purposes, like including them in your conference write up ;)
+
+Thanks to `Dennis Dawson`_ for doing them this year!
+
+.. _full resolution sketchnotes: https://flic.kr/ps/
```
Comment (thatdocslady) [inline docs/conf/atlantic/2023/news/thank-you-recap.rst]:
```suggestion
.. _full resolution sketchnotes: https://flic.kr/s/aHBqjAVj5o
```

File: docs/conf/atlantic/2023/news/thank-you-recap.rst [hunk 16]
```diff
@@ -0,0 +1,122 @@
...
+
+.. _monthly newsletter: http://writethedocs.org/newsletter/
+
+Local Meetups
+-------------
+
+We hope you continue to `build your local or online communities`_ and keep in touch with other documentarians. 
+We have a number of meetups in cities across the country, and if you don’t see your city listed consider `starting`_ a meetup!
+
+Our `Quorum Meetups`_ are special meetups, which are virtual and target large geographic areas. There’s already two, covering most everyone in the US or in a similar timezone.
```
Comment (thatdocslady) [inline docs/conf/atlantic/2023/news/thank-you-recap.rst]: I thought that we are phasing out the quorum meetups, or that they're taking a break, or something like that? cc @rosewms for input

File: docs/conf/atlantic/2023/news/thank-you-recap.rst [hunk 17]
```diff
@@ -0,0 +1,122 @@
...
+
+To ensure that our CoC was visible and accessible, we took a number of specific steps:
+
+- Every participant agreed to the Code of Conduct in advance during registration.
+- The CoC, or a summary, was included on the conference website and Hopin Reception
+- The CoC was repeatedly mentioned in our conference introduction and announcements.
+- We specifically stressed that the CoC applies to all conference spaces including the social event.
+- We encouraged all attendees to report any incident, even if they were not sure whether it was a violation.
+
+None of our organizers were approached about any CoC concerns at this event.
```
Comment (thatdocslady) [inline docs/conf/atlantic/2023/news/thank-you-recap.rst]:
```suggestion
None of our organizers were approached about any CoC concerns related to this event.
```

Comment (thatdocslady) [reply docs/conf/atlantic/2023/news/thank-you-recap.rst:106]: We did get one report about a talk title during the conf, but it was about a different conf :)

File: docs/conf/atlantic/2023/news/thank-you-recap.rst [hunk 18]
```diff
@@ -0,0 +1,122 @@
...
+
+We hope you continue to `build your local or online communities`_ and keep in touch with other documentarians. 
+We have a number of meetups in cities across the country, and if you don’t see your city listed consider `starting`_ a meetup!
+
+Our `Quorum Meetups`_ are special meetups, which are virtual and target large geographic areas. There’s already two, covering most everyone in the US or in a similar timezone.
+
+.. _build your local or online communities: http://www.writethedocs.org/meetups/
+.. _starting: http://www.writethedocs.org/organizer-guide/meetups/starting/
+.. _Quorum Meetups: https://www.writethedocs.org/organizer-guide/meetups/quorum/
+
```
Comment (thatdocslady) [inline docs/conf/atlantic/2023/news/thank-you-recap.rst:88]: Would it make sense to add a section here about 2024 conferences, or does it deserve its own blog post at some point?

Comment (mxsasha) [reply docs/conf/atlantic/2023/news/thank-you-recap.rst:88]: Let's make a separate blog post once we have pinned down some more detail.

**Pull request merged**